webapp/src/ts/modals/edit-user/edit-user-settings.component.html

```diff
@@ -0,0 +1,49 @@
+<mm-modal
+  status="status"
```

Comment:
Change to  `[status]="status"`, we need one-way binding here to set the status.

Reply:
Right. Fixed.